File: day002/Day002.java
public class Day002 {
	public static void main(String[] args) {
		java.util.Scanner kb = new java.util.Scanner(System.in);
		int x, y;
		System.out.println("Bir sayı giriniz: ");
		x = kb.nextInt();
		System.out.println("Bir sayı daha giriniz: ");
		y = kb.nextInt();
		System.out.println("Girdiğiniz sayıların toplamı: " + (x+y)); 
	}
}

File: day002/day002.py
while True:
    try:
        x = int(input("Bir sayı giriniz: "))
        y = int(input("Bir sayı daha giriniz: "))
        print("İşlem başarılı, girdiğiniz sayıların toplamı:", x+y)
        break

    except Exception:
        print("Lütfen tam sayı değeri giriniz!")
        continue
